frontend/src/components/ShootWorkers/VolumeType.vue

```diff
@@ -61,15 +62,58 @@ export default {
     volumeTypes: {
       type: Array,
       default: () => []
+    },
+    cloudProfileName: {
+      type: String
     }
   },
   data () {
     return {
-      validationErrors,
-      valid: undefined
+      valid: undefined,
+      internalIops: undefined
     }
   },
+  validations () {
+    return this.validators
+  },
   computed: {
+    ...mapGetters([
+      'cloudProfileByName'
+    ]),
+    validators () {
+      return {
+        worker: {
+          volume: {
+            type: {
+              required
+            }
+          }
+        },
+        internalIops: {
+          required: requiredIf(function () {
+            return this.isAWS && this.worker.volume.type === 'io1'
```

Comment:
Why is `this` the component here?

Reply:
You mean why does it work as it is no arrow function? Actually a good question, I don't know. But we can change it to an arrow function if you want.

Reply:
=>changed to arrow function